operator/image.yaml

```diff
@@ -31,6 +33,8 @@ modules:
         ref: sprint-28
     - name: modules
       path: modules
+    - name: manifests
+      path: manifests
```

Comment:
After creating an "olm" module ( refer to @ppatierno 's comment ), we can install it using
```
-name: olm
```
the same way we install the "operator" module [1]. We won't need these two lines


[1] https://github.com/jboss-container-images/amqstreams-1-openshift-image/blob/f785f9801f25ac7dc95c7e4e8d884c4e9ead4ae6/operator/image.yaml#L43